spec/presenters/error_message/presenter_spec.rb

```diff
@@ -143,6 +143,16 @@
       it { is_expected.to eq('Foo bar') }
     end
 
+    xcontext 'with a nested error that is generated by BaseSubModelValidator' do
```

Comment:
extra 'x'

Reply:
That's deliberate, adding `x` before the test skips it. Those two tests don't actually do anything until this code is used to migrate a form, so I've disabled them for now (maybe I should have removed them completely 🤷 ). 

My thinking is that once the `graduated_fee` form (for example) has been migrated we can remove the `x` and the test will run and pass. (If another form is migrated first then the content of the test will need to be tweaked a bit).

I mention this in the PR description but I can add comments to the tests for more clarity if it would be useful?

Reply:
Ah, thanks and apologies for not reading the description properly. I hope we don't forget to update this file during the migrated form implementation.

Reply:
No worries. I agree there's a small risk that it gets missed. I'll be checking your PRs with eagle eyes 😀

Reply:
Are they expected to fail at the moment? If you add `pending` instead of the `x` then they will report as a pass if they fail and as a failure if they pass. This means that when the behaviour is fixed the tests get highlighted so that they can be updated.

Reply:
I did not know about `pending`. That does exactly what I need. Every day's a school day!